docs/maps/import-geospatial-data.asciidoc

```diff
@@ -0,0 +1,30 @@
+[role="xpack"]
+[[import-geospatial-data]]
+== Import geospatial data
+
+In order to use your geospatial data in the Elastic Stack, it must be indexed as {ref}/geo-point.html[geo_point] or {ref}/geo-shape.html[geo_shape].
```

Comment:
The intro needs some work.  Are CSV Upload and GeoJSON upload two different ways to get your data into Elasticsearch and have it set up correctly?

Maybe this for the first sentence?

Geospatial data in the Elastic Stack must be indexed as a geo_point or geo_shape.

Reply:
> Are CSV Upload and GeoJSON upload two different ways to get your data into Elasticsearch and have it set up correctly?

Yes